modules/nix/default.nix

```diff
@@ -823,8 +823,58 @@ in
       ]);
     users.knownGroups = mkIf cfg.configureBuildUsers [ "nixbld" ];
 
+    # The Determinate Systems installer puts user‐specified settings in
+    # `/etc/nix/nix.custom.conf` since vX.YY.Z. Supplement the
```

Comment:
I'm guessing this is meant to have an actual version number

Reply:
Whoops, fixed.